packages/screens/FeedPostView/components/FeedPostArticleView.tsx

```diff
@@ -245,45 +250,47 @@ export const FeedPostArticleView: FC<{
             }}
           >
             <SocialCardWrapper post={localPost} refetchFeed={refetchPost}>
-              <SocialCardHeader
-                authorAddress={authorAddress}
-                authorId={localPost.authorId}
-                createdAt={post.createdAt}
-                authorMetadata={authorNSInfo?.metadata}
-              />
-
-              <SpacerColumn size={1.5} />
-
-              {/*====== Card Content */}
-              <View>
-                {!!metadataToUse?.title && (
-                  <BrandText style={{ marginBottom: layout.spacing_x1 }}>
-                    {metadataToUse.title}
-                  </BrandText>
-                )}
-                {!!coverImage && (
+              {/*========== Article title, author info and cover image */}
+              {!!coverImage && (
+                <>
                   <Image
                     source={{ uri: web3ToWeb2URI(coverImage.url) }}
                     resizeMode="cover"
                     style={{
                       width: "100%",
-                      height: ARTICLE_COVER_IMAGE_HEIGHT,
-                      marginBottom: layout.spacing_x1_5,
+                      aspectRatio: ARTICLE_COVER_IMAGE_RATIO,
+                      // height: ARTICLE_COVER_IMAGE_MAX_HEIGHT/1.6,
+                      maxHeight: ARTICLE_COVER_IMAGE_MAX_HEIGHT,
                     }}
                   />
-                )}
+                  <SpacerColumn size={3} />
+                </>
+              )}
+
+              {!!metadataToUse?.title && (
+                <BrandText>{metadataToUse.title}</BrandText>
+              )}
+
+              <SpacerColumn size={1.5} />
+              <SocialCardHeader
+                authorAddress={authorAddress}
+                authorId={localPost.authorId}
+                createdAt={post.createdAt}
+                authorMetadata={authorNSInfo?.metadata}
+              />
+              <SpacerColumn size={1.5} />
 
-                <RichText
-                  initialValue={metadataToUse.message}
-                  isPostConsultation
-                  audioFiles={audioFiles}
-                  postId={postId}
-                  authorId={authorId}
-                />
-              </View>
+              {/*========== Article content */}
+              <RichText
+                initialValue={metadataToUse.message}
+                isPostConsultation
+                audioFiles={audioFiles}
+                postId={postId}
+                authorId={authorId}
+              />
               <SpacerColumn size={1.5} />
 
-              {/*====== Card Actions */}
+              {/*========== Actions */}
```

Comment:
I don't understand why we have some comments that start with `=` ^^

Reply:
Just to make it more visible, it's just a decoration, no guideline for this